src/ray/object_manager/object_directory.h

```diff
@@ -0,0 +1,135 @@
+#ifndef RAY_OBJECT_DIRECTORY_H
+#define RAY_OBJECT_DIRECTORY_H
+
+#include <memory>
+#include <vector>
+#include <unordered_set>
+#include <unordered_map>
```

Comment:
"Within each section the includes should be ordered alphabetically." https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
yeah, we're on the same page w.r.t. all the include comments.